scripts/metadataProvider/createPseudoModulesInfo.ts

```diff
@@ -0,0 +1,182 @@
+import {pipeline} from "node:stream/promises";
+import {Extract} from "unzip-stream";
+import MetadataProvider from "./MetadataProvider.js";
+import {writeFile, readdir} from "node:fs/promises";
+import {createRequire} from "module";
+const require = createRequire(import.meta.url);
+
```

Comment:
remove empty line / sorting the imports alphabetically would also be an option

Reply:
Done. However, I have left the empty line as the first ones are module imports and the line below is a type import and I wanted to distinguish them